### PR TITLE
Updates for Samesite Cookies

### PR DESCRIPTION
**Pull request by aarongodin**
## Changes

- Adds support for Samesite cookie changes coming to browsers.
- Updates the readme to be more direct about what the current state of the extension is, as well as current state of authorization in Auth0. (with better instructions on running the extension locally)

## Testing

You will need to enable the samesite flags to test the changes.

In Firefox, open a new tab and navigate to `about:config`.

Use the following flags set to true:

* `network.cookie.sameSite.noneRequiresSecure`
* `network.cookie.sameSite.laxByDefault`

## References (internal)

https://auth0team.atlassian.net/browse/AUTHZ-1413

### CODE REVIEW

File: build/webpack/server.js [hunk 1]
```diff
@@ -17,15 +17,15 @@ logger.error = function error() {
 };
 
 const options = {
-  publicPath: 'http://localhost:3000/app/',
+  publicPath: 'http://localhost:3001/app/',
```
Comment (jhiner) [inline build/webpack/server.js]: Can you explain the different ports 3001 vs 3000? I don't follow 🤔

File: webtask.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "title": "Auth0 Authorization",
   "name": "auth0-authz",
-  "version": "2.6.5",
-  "preVersion": "2.5.2",
+  "version": "2.7.0",
+  "preVersion": "2.6.5",
```
Comment (jhiner) [inline webtask.json]: What does the preVersion do again? I don't recall having to change this in the other extension.

File: build/webpack/server.js (hunk 1 above)
Comment (jhiner) [reply build/webpack/server.js]: (question applies to all the different places these are changed throughout the PR)

Comment (ubenzer) [reply build/webpack/server.js]: In the development mode, the webask development server and the proxy port (the actual hapi server port) were reversed. Due to this, webtask was unable to proxy requests to hapi, which was making the development mode broken. I reversed them.

Comment (ubenzer) [reply build/webpack/server.js]: Note: This shouldn't affect production in any way.

File: webtask.json (hunk 2 above)
Comment (ubenzer) [reply webtask.json]: I am not really sure. I've checked with DAE and didn't see it there either. I'll get rid of it.

Comment (ubenzer) [reply webtask.json]: https://github.com/auth0/auth0-authorization-extension/pull/260/commits/a4b5fa0c5233cdab72b48c589ed62cd2015848eb

**Pull request merged**